threatminer/plugin.spec.yaml

```diff
@@ -279,13 +278,13 @@ actions:
         required: false
         description: Response
   email_domain:
-    title: Email (Reverse WHOIS) - Domain
+    title: Email Reverse WHOIS - Domain
```

Comment:
Title change is a full version bump for the plugin. As it changes how its accessed in insightconnect.

Reply:
Ah good call @lmilby-r7 .  Ultimately the title change was because of some tooling deficiencies that I need to get a ticket opened for.  Appears the `(` and`)`cause generation problems